packages/@jsii/java-runtime/pom.xml.t.js

```diff
@@ -119,6 +120,13 @@ process.stdout.write(`<?xml version="1.0" encoding="UTF-8"?>
             <version>\${mockito.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- https://mvnrepository.com/artifact/javax.annotation/javax.annotation-api -->
+        <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>\${javax-annotations.version}</version>
+        </dependency>
```

Comment:
This improves combat with "more recent" JDKs (where the `@javax.annotations.*` was moved to a different package).